### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by Tomanhez**

### CODE REVIEW

**Review by lchrusciel — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -57,6 +57,13 @@ jobs:
                     mysql version: "${{ matrix.mysql }}"
                     mysql root password: "root"
 
+            -
+                name: Install wkthtmltopdf
+                run: |
+                    sudo apt-get install -y xfonts-base xfonts-75dpi
+                    wget https://github.com/wkhtmltopdf/packaging/releases/download/0.12.6-1/wkhtmltox_0.12.6-1.focal_amd64.deb
+                    sudo dpkg -i wkhtmltox_0.12.6-1.focal_amd64.deb
```
[the remark this answers is not in the document]
Comment (arti0090) [reply .github/workflows/build.yml:65]: FYI my comment is no blocker - just asking 🙃

File: composer.json [hunk 2]
```diff
@@ -25,11 +25,11 @@
         "sensiolabs/security-checker": " ^6.0",
         "sylius/sylius": "dev-master",
         "symfony/flex": "^1.7",
-        "sylius/invoicing-plugin": "^0.14",
-        "sylius/refund-plugin": "^1.0.0-RC.10",
+        "sylius/invoicing-plugin": "^0.17",
```
Comment (GSadee) [inline composer.json:28]: To not to block on only one version:
```suggestion
        "sylius/invoicing-plugin": "~0.17",
```

File: .github/workflows/build.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Tomanhez) [reply .github/workflows/build.yml:65]: I tested it and `sudo apt-get install wkhtmltopdf -y` doesn't install wkhtmltopdf

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Tomasz! :1st_place_medal:

**Pull request merged**